src/imp/tracer_imp.js

```diff
@@ -234,6 +235,9 @@ export default class Tracer extends opentracing.Tracer {
         this.addOption('log_field_key_hard_limit',   { type: 'int',     defaultValue: 256 });
         this.addOption('log_field_value_hard_limit', { type: 'int',     defaultValue: 1024 });
 
+        // Meta Event reporting options
+        this.addOption('meta_event_reporting', { type: 'bool', defaultValue: false });
```

Comment:
I thought we discussed having an OPT-OUT here, vs. an OPT-IN, since the satellite will only return this when we want it on. It's going to be more work to convince users to turn this on if we require OPT-IN.

Reply:
Oh, I see we have an OPT-IN in Go, too. I don't think this is what we want, but it's consistent.

Reply:
We want opt-in so that a blind update doesn't turn on meta event reporting.

Reply:
(I still feel we want OPT-OUT in the tracers. If we want to do OPT-IN, let's use a project-level setting.)